packages/lambda-powertools-pattern-basic/index.js

```diff
@@ -12,7 +13,10 @@ if (!process.env.DATADOG_PREFIX) {
   process.env.DATADOG_PREFIX = FUNCTION_NAME + '.'
 }
 
-process.env.DATADOG_TAGS = `awsRegion:${AWS_REGION},functionName:${FUNCTION_NAME},functionVersion:${FUNCTION_VERSION},environment:${ENV}`
+process.env.DATADOG_TAGS = supplementCsv({
+  existing: process.env.DATADOG_TAGS,
+  additional: `awsRegion:${AWS_REGION},functionName:${FUNCTION_NAME},functionVersion:${FUNCTION_VERSION},environment:${ENV}`
```

Comment:
since we're in control of the `additional` tags, would it be easier if we pass in a map instead of a CSV string?

Reply:
e.g. 
```
process.env.DATADOG_TAGS = supplementCsv({
  existing: process.env.DATADOG_TAGS,
  additional: {
    awsRegion: AWS_REGION,
    functionName: FUNCTION_NAME,
    functionVersion: FUNCTION_VERSION,
    environment: ENV
  }
})
```

Reply:
The reason for having it as a csv string was to minimise the impact of change if it is rolled out to other patterns...  but it is much easier to construct a map than build a string so I'm happy to make that change.